src/containers/Tables/OwnedDomainsTable/OwnedDomainsTable.tsx

```diff
@@ -217,26 +206,17 @@ const OwnedDomainTables: React.FC<OwnedDomainTableProps> = ({ onNavigate }) => {
 	return (
 		<>
 			{overlays()}
-			{isTableLoading && (
-				<>
-					<p className={styles.Message}>Loading Your Domains</p>
-					<Spinner style={{ margin: '8px auto' }} />
-				</>
-			)}
-			<DomainTable
-				className={styles.Reset}
-				domains={owned}
-				isButtonActive={isButtonActive}
-				filterOwnBids={true}
-				isRootDomain={false}
-				rowButtonText={'View Bids'}
-				onLoad={tableLoaded}
+			<GenericTable
+				alignments={[0, 0, 1, 1, 1]}
+				headers={['', 'DOMAIN', 'HIGHEST BID', '# OF BIDS', '']}
+				data={owned}
+				loadingText={'Loading Domains'}
+				rowComponent={OwnedDomainTableRow}
+				gridComponent={OwnedDomainTableCard}
+				rowClick={rowClick}
 				onRowButtonClick={viewBid}
-				onRowClick={rowClick}
-				isGridView={isGridView}
-				setIsGridView={(grid: boolean) => setIsGridView(grid)}
+				filterOwnBids={true}
 				userId={account || undefined}
```

Comment:
Minor: `account` will be undefined if the user is disconnected so you don't need the `|| undefined` - just letting you know for future reference